app/Console/Commands/DevCheckCommand.php

```diff
@@ -52,6 +52,7 @@ public function __construct()
         $this->addOption('full', null, InputOption::VALUE_NONE);
         $this->addOption('os-modules-only', null, InputOption::VALUE_NONE);
         $this->addOption('commands', 'c', InputOption::VALUE_NONE);
+        $this->addOption('debug', 'd', InputOption::VALUE_NONE);
```

Comment:
Do we want to match device:poll and use -vvv?

Reply:
Seems that it is already there so nothing to change in the options I suppose. Just extend the code, if necessary, to show more if verbosity level is 3